autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    DB_PASS = 'foo123'
</pre><p>Make sure to run <code>black-nb -l 100 {notebook-name}.ipynb</code> to apply standard Python code formatting, which includes using double quotes instead of single quotes. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I think I got all the ' manually.  It'd probably still be good for me to understand how to run the linter properly though.  Advice appreciated there.</p>

Reply:
<p>I was able to get the black-nb formatting to work locally on a copied version of the ipynb file. I ran <code>black-nb -l 100 sagemaker_autopilot_neo4j_portfolio_churn.ipynb</code>. You can omit the tox check.  When black-nb succeeded, the message was:</p><pre class="ql-syntax" spellcheck="false">% black-nb -l 100 sagemaker_autopilot_neo4j_portfolio_churn.ipynb
reformatted sagemaker_autopilot_neo4j_portfolio_churn.ipynb

&nbsp; &nbsp; 17 cells reformatted, 15 cells left unchanged.

All done! ✨ 🍰 ✨

1 file reformatted.
</pre>

Reply:
Got it.  Thanks.  Ran and reformatted.